README.md

```diff
@@ -14,6 +14,10 @@ $ ./develop_server.sh start
 
 The contents should be visible at [http://localhost:8000/blog/](http://localhost:8000/blog/)
 
+## Contributing to the Blog site
+
+Please read our [Contribution Guidelines](CONTRIBUTING.md) to Contribute to the Blog site
```

Comment:
Capital letter in the middle of sentences doesn't look good anywhere and neither does grammatical errors in README.md. 

Rephrasing 

**Want to contribute?**

Thank you for your you interest in contributing to PyCon India blog site, please do read our [Contribution Guidelines](CONTRIBUTING.md) to get started. Submit an issue in the Issues section for reporting/fixing bugs and to request new features.